source/user-manual/capabilities/system-inventory/viewing-system-inventory-data.rst

```diff
@@ -755,7 +755,7 @@ Follow the steps below to query the system inventory indices from the command li
 Using the Wazuh server API
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can query the Wazuh inventory data using the `Wazuh server API <https://documentation.wazuh.com/current/user-manual/api/reference.html#tag/Syscollector>`__, which retrieves nested data in JSON format. You can use the Wazuh server API GUI on the dashboard or a command-line tool like ``cURL`` to query the inventory database of a Wazuh agent.
+You can query the Wazuh inventory data using the :doc:`Wazuh server API </user-manual/api/reference>`, which retrieves nested data in JSON format. You can use the Wazuh server API GUI on the dashboard or a command-line tool like ``cURL`` to query the inventory database of a Wazuh agent.
```

Comment:
We need to ask the author where these links must point to if we want to update them